x-pack/plugins/observability_solution/logs_data_access/kibana.jsonc

```diff
@@ -0,0 +1,16 @@
+{
+  "type": "plugin",
+  "id": "@kbn/logs-data-access-plugin",
+  "owner": "@elastic/obs-ux-logs-team",
```

Comment:
this should probably be added to the codeowners file too

Reply:
done https://github.com/elastic/kibana/pull/181823/commits/ae2aa1ad02798afe11dba25d7ccd60d6b4d64d60